### DISCUSSION TITLE
PurgeCSS > add classes to safelist / whitelist

### DISCUSSION THREAD

**Discussion by keeslina** [tailpress/tailpress#63] [Q&A] [answered]
Hi, 

i'm trying to add classes to a safelist for purgecss, like this:

```
purge: {
        content: [
            './theme/*.php',
            './theme/**/*.php',
            './resources/css/*.css',
            './resources/js/*.js',
            './safelist.txt'
        ],
        options: {
            safelist: [
                'bg-lichtroze',
                'bg-roze',
                'bg-paars'
            ]
        }
    },
```

This is because these classes are added as a php value from a Wordpress ACF field, like this:
`<?php the_field('background-color'); ?>`

 So they are stored in the database..

The classes don't exist in my code and are not found by purgecss..

The safelist works, but only  with JIT mode turned off.  
Any idea how i can get this to work with JIT mode turned on?

Same issue is described here: https://github.com/tailwindlabs/tailwindcss/issues/2829

**Comment by jeffreyvr** [comment]
You can add those class names to the `safelist.txt` file. That should work wit JIT as well.

**Comment by keeslina** [comment, marked as answer]
Hi Jeffrey,

Just found out that this also seems to work:

```
purge: {
               safelist: [
            'bg-lichtroze',
            'bg-roze',
            'bg-paars'
        ]
    },
```


So not wrapping the safelist in an options container.
